### ISSUE TITLE
Mod conflict with "Enhanced Block Entities" causes signs to still render while displaying an image

### ISSUE THREAD

**Issue by EnderMelody** [Nettakrim/Signed-Paintings#6]
minecraft: 1.20.1
quilt: 0.20.2 
signed paintings: 1.0.2
enhanced block entities: 1.0.4

using the mod https://modrinth.com/mod/ebe causes signs that are rendering images with this mod to still be rendered. This can be solved by turning off the enhanced signs option however it removes the benefit of the ebe mod for signs doing this.

**Comment by Nettakrim** [comment]
huh as in both the image and sign appear?

that's odd as I had specifically made the mods compatible as per issue #1

given your list, the only possible differences I can see would be either ebe updated, or its an issue with quilt

**Comment by EnderMelody** [comment]
yes that is the exact problem though the image still renders correctly as well.
seems like ebe must have updated then as even with fabric it is still rendering the sign.

![image](https://github.com/Nettakrim/Signed-Paintings/assets/12649140/8ab6c3e4-6c52-48e8-ad40-db5e41d393fd)

**Comment by Nettakrim** [comment]
huh weird it is in fact also happening for me on my main install, even though i was pretty sure i had tested it

**Comment by Nettakrim** [comment]
yeah odd its stopping text rendering but not not the sign itself

**Comment by Nettakrim** [comment]
for some reason ebe doesnt even want to load in my dev environment so not much i can do for the time being

**Comment by Nettakrim** [comment]
hmm yeah even [the version](https://github.com/Nettakrim/Signed-Paintings/actions/runs/6251353407) from the moment i allegedly fixed it has that issue - cant be bothered to figure out the issue rn as its not too important
and the code for ebe is mystifying

**Comment by EnderMelody** [comment]
yup no rush since you can just disable the sign model for ebe and you wont ever really have that many signs that it should matter. just thought you should know that there's a conflict.

**Comment by Nettakrim** [comment]
i looked at this again and i still cant figure out where ebe is rendering their models lol

**Comment by PurpBatBoi** [comment]
october update, still happening

**Comment by Nettakrim** [comment]
yeah the way ebe works i cant really do much about it, i should probably have a message show up in chat though when it detects ebe
